src/toolkits/object_detection/one_shot_object_detection/one_shot_object_detector.cpp

```diff
@@ -149,14 +150,14 @@ create_synthetic_image_from_background_and_starter(const flex_image &starter,
                        reinterpret_cast<const boost::gil::rgba8_pixel_t *>(
                            rgba_flex_image.get_image_data()),
                        rgba_flex_image.m_channels *
-                           rgba_flex_image.m_width  // row length in bytes
+                           rgba_flex_image.m_width // row length in bytes
```

Comment:
Any reason to remove the space here? I recommend we just use `clang-format` to resolve formatting decisions and be done with it.

In this case, I believe clang-format should leave two spaces: https://google.github.io/styleguide/cppguide.html#Implementation_Comments